src/index.ts

```diff
@@ -31,6 +32,18 @@ export const handleBase64Json = (tokenUri: string): Reason[] => {
   return reasons;
 };
 
+export const handleHttp = async (tokenUri: string): Promise<Reason[]> => {
+  let reasons: Reason[] = [Reasons.tokenUriIsHttp];
+
+  let res: Metadata = await axios.get(tokenUri);
+
+  if (res.image && isTokenUriHttp(res.image)) {
+    reasons = [...reasons, Reasons.imageUriIsHttp];
+  }
```

Comment:
As a followup, what should the evaluation be if the image is IPFS? Having the image as IPFS is nice, but does that really give you any extra points if it's sitting at a rando URL?